coding_guidelines_c++.md

```diff
@@ -78,7 +72,7 @@ OS X:
 
 Or consider installing `vera++` from the [binary packages](https://bitbucket.org/verateam/vera/wiki/Installation).
 
-Finally, we let [cppcheck](http://cppcheck.sourceforge.net/) statically analyse the committed files and check for severe errors:
+Finally, we let [cppcheck](http://cppcheck.sourceforge.net/) statically analyse the committed files and check for severe errors. We require cppcheck version 1.69.
 
```

Comment:
I guess, we can add 'or later' to cppcheck, as the results here are not critical for success of TravisCI tasks.